tools/gradle-plugins/gradle-curiostack-plugin/src/main/java/org/curioswitch/gradle/plugins/gcloud/CurioDatabasePlugin.java

```diff
@@ -68,8 +68,8 @@ private static void addTasks(Project project) {
     if (devAdminPassword != null) {
       generateDevDbDockerfile.environmentVariable("MYSQL_PASSWORD", devAdminPassword);
     }
-    generateDevDbDockerfile.environmentVariable("MYSQL_RANDOM_ROOT_PASSWORD", "yes");
-    generateDevDbDockerfile.environmentVariable("MYSQL_ONETIME_PASSWORD", "yes");
+    // Root privilege need to be exposed to create users other than MYSQL_USER.
+    generateDevDbDockerfile.environmentVariable("MYSQL_ROOT_PASSWORD", "root");
```

Comment:
This shouldn't be such a simple password - let's go ahead and use the same devAdminPassword for the user and root.